src/lib/RequestTypes.ts

```diff
@@ -117,15 +118,35 @@ export interface LoginResult {
     }>;
 }
 
+export interface LogoutRequest {
+    kind?: RequestType.LOGOUT;
+    appName: string;
+    keyId: string;
+    keyLabel?: string;
```

Comment:
`keyLabel` should not be accepted from outside, but just be filled in by the AccountsManager.